tools/generator/Java.Interop.Tools.Generator.ObjectModel/Symbols/ArraySymbol.cs

```diff
@@ -106,8 +106,8 @@ public string[] PostCall (CodeGenerationOptions opt, string var_name)
 		{
 			string native_name = opt.GetSafeIdentifier (TypeNameUtilities.GetNativeName (var_name));
 			string[] result = new string [4];
-			result [0] = String.Format ("if ({0} != null) {{", var_name);
-			result [1] = String.Format ("\tJNIEnv.CopyArray ({0}, {1});", native_name, var_name);
+			result [0] = String.Format ("if ({0} != null) {{", opt.GetSafeIdentifier (var_name));
```

Comment:
Presumably the *caller* of `PostCall()` is also using `opt.GetSafeIdentifier()` -- *something* somewhere else must be, because this code is just the error checking code! -- so would it instead be better if we called `opt.GetSafeIdentifier()` *once* and passed that value around to all the methods that needed it, instead of duplicating `opt.GetSafeIdentifier()` invocations?

Reply:
I think that is a much trickier and dangerous change to get correct.  For example the previous line is:
```
string native_name = opt.GetSafeIdentifier (TypeNameUtilities.GetNativeName (var_name));
```
If we're passing in a `var_name` that has already been mangled then getting the native name from it will probably need to be reworked, etc.